tests/core/test_snapshot.py

```diff
@@ -1746,3 +1746,10 @@ def test_missing_intervals_node_start_end(make_snapshot):
     assert missing_intervals([snapshot], start="2024-03-01", end=to_datetime("2024-03-10")) == {}
     assert missing_intervals([snapshot], start="2024-03-13", end="2024-03-14") == {}
     assert missing_intervals([snapshot], start="2024-03-14", end="2024-03-30") == {}
+
+
+def test_external_model_audits(sushi_context):
```

Comment:
I think we also need a test in the `test_snapshot_evaluator.py` to ensure that audits are actually called for external model snapshots.